examples/inference/inpainting.py

```diff
@@ -0,0 +1,182 @@
+import inspect
+from typing import List, Optional, Union
+
+import numpy as np
+import torch
+
+import PIL
+from diffusers import AutoencoderKL, DDIMScheduler, DiffusionPipeline, PNDMScheduler, UNet2DConditionModel
+from diffusers.pipelines.stable_diffusion import StableDiffusionSafetyChecker
+from tqdm.auto import tqdm
+from transformers import CLIPFeatureExtractor, CLIPTextModel, CLIPTokenizer
+
+
+def preprocess(image):
```

Comment:
is this function used? Think we can delete it no?

Reply:
If it needs to be used for the image, I'd indeed put it along-side pre-processing the mask

Reply:
Yes, pulling image preprocessing inside seems best